nautobot_netbox_importer/management/commands/import_netbox_objectchange_json.py

```diff
@@ -0,0 +1,152 @@
+        if not entry["fields"]["user"]:
+            error_message = f'Username {entry["fields"]["user_name"]} not present in DB.'
+            if error_message not in used_error_messages:
+                used_error_messages.add(error_message)
+                self.logger.error(error_message)
+            return
+
+        if (
+            not options["dry_run"]
+            and ObjectChange.objects.filter(
+                request_id=entry["fields"]["request_id"], time=entry["fields"]["time"]
+            ).count()
+            == 0
+        ):
+
+            obj = ObjectChange.objects.create(**entry["fields"])
+            obj.time = entry["fields"]["time"]
+            obj.full_clean()
+            obj.save()
+
+    def handle(self, *args, **options):
+        """Handle execution of the import_netbox_json management command."""
+        validate_netbox_version(options["netbox_version"])
+
+        self.logger, _ = initialize_logger(options)
+
+        self.logger.info("Loading NetBox JSON data into memory...", filename={options["json_file"].name})
+        try:
+            no_objectchange_data = json.load(options["json_file"])
+        except ValueError as json_error:
+            raise CommandError(  # pylint: disable=raise-missing-from
+                f"{options['json_file']} is not a valid JSON file: {json_error}"
+            )
+        if not isinstance(no_objectchange_data, list):
+            raise CommandError(f"Data should be a list of records, but instead is {type(no_objectchange_data)}!")
+        self.logger.info("JSON data loaded into memory successfully.")
+
+        self.logger.info("Creating NetBox ContentType mapping...")
+        for entry in no_objectchange_data:
+            if entry["model"] == "contenttypes.contenttype":
+                self.netbox_contenttype_mapping[entry["pk"]] = (
+                    entry["fields"]["app_label"],
+                    entry["fields"]["model"],
+                )
+        # Freeing some memory once the object is processed
+        no_objectchange_data = None
+
+        self.logger.info("Creating Nautobot ContentType mapping...")
+        for entry in ContentType.objects.all():
+            self.nautobot_contenttype_mapping[(entry.app_label, entry.model)] = entry.id
+
+        self.logger.info(
+            "Loading ObjectChange NetBox JSON data into memory...", filename={options["objectchange_json_file"].name}
+        )
+        try:
+            objectchange_data = json.load(options["objectchange_json_file"])
+        except ValueError as json_error:
+            raise CommandError(  # pylint: disable=raise-missing-from
+                f"{options['objectchange_json_file']} is not a valid JSON file: {json_error}"
+            )
+
+        # used_error_messages is used to avoid repeating the same error message for each entry
+        used_error_messages = set()
+        for entry in ProgressBar(objectchange_data):
```

Comment:
To hide the progressbar when run with verbosity=0, matching the other importer command:

```suggestion
        for entry in ProgressBar(objectchange_data, verbosity=options["verbosity"]):
```